repose-aggregator/components/filters/keystone-v2-filter/src/test/scala/org/openrepose/filters/keystonev2/KeystoneV2FilterRcnTest.scala

```diff
@@ -102,24 +108,39 @@ class KeystoneV2FilterRcnTest extends FunSpec
         response.setStatus(responseStatus)
         val filterChain = new MockFilterChain()
 
-        when(mockAkkaServiceClient.get(
-          mockitoEq(s"$TOKEN_KEY_PREFIX$VALID_TOKEN"),
-          mockitoEq(s"$identityServiceUri$TOKEN_ENDPOINT/$VALID_TOKEN${appendRcnParameter(applyRcnRoles)}"),
-          argThat(hasEntry(CommonHttpHeader.AUTH_TOKEN, VALID_TOKEN)),
-          anyBoolean()
-        )).thenReturn(new ServiceClientResponse(SC_OK, validateTokenResponse(userId = VALID_USER_ID)))
-        when(mockAkkaServiceClient.get(
-          mockitoEq(s"$ENDPOINTS_KEY_PREFIX$VALID_TOKEN"),
-          mockitoEq(s"$identityServiceUri${ENDPOINTS_ENDPOINT(VALID_TOKEN)}${appendRcnParameter(applyRcnRoles)}"),
-          argThat(hasEntry(CommonHttpHeader.AUTH_TOKEN, VALID_TOKEN)),
-          anyBoolean()
-        )).thenReturn(new ServiceClientResponse(SC_OK, endpointsResponse()))
-        when(mockAkkaServiceClient.get(
-          mockitoEq(s"$GROUPS_KEY_PREFIX$VALID_USER_ID"),
-          mockitoEq(s"$identityServiceUri${GROUPS_ENDPOINT(VALID_USER_ID)}"),
-          argThat(hasEntry(CommonHttpHeader.AUTH_TOKEN, VALID_TOKEN)),
-          anyBoolean()
-        )).thenReturn(new ServiceClientResponse(SC_OK, groupsResponse()))
+        when(mockHttpClient.execute(any[HttpUriRequest], any[HttpContext]))
+            .thenAnswer(new Answer[CloseableHttpResponse] {
```

Comment:
I went to write a custom matcher and found this:
https://github.com/rackerlabs/repose/blob/master/repose-aggregator/commons/commons-test/src/main/scala/org/openrepose/commons/test/MockitoAnswers.scala

At a minimum, I will use that to clean up these answers a bit. Hopefully the custom matcher will work out better, though.